Formula/dockerize.rb

```diff
@@ -1,8 +1,8 @@
 class Dockerize < Formula
   desc "Utility to simplify running applications in docker containers"
-  homepage "https://github.com/jwilder/dockerize"
-  url "https://github.com/jwilder/dockerize/archive/v0.6.1.tar.gz"
-  sha256 "c21cea3e6bb33a2e280c28d3521b8f177c78e875b475763fcb9bd7a545e21688"
+  homepage "https://github.com/powerman/dockerize"
+  url "https://github.com/powerman/dockerize/archive/refs/tags/v0.16.0.tar.gz"
+  sha256 "b6507b24714dee193bd04be18fe8658531d2b0c0b9f6f060ac6bb387c736009f"
```

Comment:
Because the `jwilder/dockerize` is not maintained anymore (see [`powerman/dockerize`](https://github.com/powerman/dockerize)):

> **About this fork**: This fork is supposed to become a community-maintained replacement for [not maintained](https://github.com/powerman/dockerize/issues/19) [original repo](https://github.com/jwilder/dockerize). Everyone who has contributed to the project may become a collaborator - just ask for it in PR comments after your PR has being merged.

Reply:
The original repo doesn't seem that unmaintained though, last commit was just 4-ish months ago.

Reply:
It makes sense to update if it really is inactive, could you make an issue in the original repo asking if this fork is an actual replacement? If we can get upstream to confirm the replacement it's fine with me.

Reply:
Hi, guys.
Yes, my fork is not "official", and you can read the full story in https://github.com/powerman/dockerize/issues/19
To me, official repo still looks mostly unmaintained, but it's good to see there was some activity in Dec 21.

Reply:
If you're willing to turn this into a new `powerman-dockerize` formula I'd happily accept that @schrieveslaach

Reply:
okay, will do. 👍

Reply:
@SMillerDev, done. Do you mind to review it?